Makefile

```diff
@@ -43,6 +43,7 @@ install: all
 	+$(MAKE) -C libs install INSTALLDIR=$(INSTALLDIR_ABS)/libs
 	+$(MAKE) -C sys install INSTALLDIR=$(INSTALLDIR_ABS)/sys
 	+$(MAKE) -C tools install INSTALLDIR=$(INSTALLDIR_ABS)/tools
+	+$(CP) -r sys/cmake $(INSTALLDIR_ABS)/cmake
```

Comment:
Move this to `sys/Makefile` for consistency.

Also, can you squash all the commits and rename the resulting one to something like: `cmake: examples: Add build scripts and example`? Or maybe you can have two commits: `cmake: Add build scripts and documentation` and `examples: Add CMake example`

Reply:
I actually would prefer not to do this because `cmake` is not a subdirectory of `$(INSTALLDIR_ABS)/sys`. And yes, I'll squash now.